.github/pull_request_template.md

```diff
@@ -8,6 +8,11 @@ Relevant issues: <!-- Please add issue numbers. -->
 
 <!-- Please confirm there's no breaking change, or call our any behavior changes you think are necessary. -->
 
+## Does this change need a changelog entry in any of the crates?
```

Comment:
nit: links to those change-logs would be nice, but I bet most of contributors will know where to find them